EssentialFeed/EssentialFeedTests/Image Comment/API/ImageCommentsMapper.swift

```diff
@@ -0,0 +1,70 @@
+//
+// Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import Foundation
+
+import XCTest
+import EssentialFeed
+
+class ImageCommentMapper: XCTestCase {
```

Comment:
The convention is to name `XCTestCase` subclasses with `Tests` at the end.

Moreover, there's a typo in the name (missing an `s` in `Comments` to match the type name.)

Please, rename this class _and the file as well_.

```suggestion
class ImageCommentsMapperTests: XCTestCase {
```